app/web/src/store/func/funcs.store.ts

```diff
@@ -618,11 +786,14 @@ export const useFuncStore = () => {
 
               if (this.selectedFuncId) {
                 // only fetch if we don't have this one already in our state,
+                // or if the func kind is attribute
                 // otherwise we can overwrite functions with their previous value
                 // before the save queue is drained.
                 if (
-                  typeof this.funcDetailsById[this.selectedFuncId] ===
-                    "undefined" &&
+                  (typeof this.funcDetailsById[this.selectedFuncId] ===
```

Comment:
I think this does what we need given I'm using the FuncSaved event still when Attribute Prototypes are modified, but I'd appreciate a second opinion :)